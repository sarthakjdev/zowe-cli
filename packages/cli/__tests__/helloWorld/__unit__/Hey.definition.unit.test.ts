import { ICommandDefinition } from "@zowe/imperative";

describe("hey command definition", () => {
    it("should say hello", () => {
        const definition: ICommandDefinition = require("../../../src/helloWorld/Hey.definition");

        expect(definition).toBeDefined();

        expect(definition.aliases).toBeDefined();

        expect(definition.handler).toBeDefined();

        expect(definition.type).toBeDefined();

        expect(definition.summary).toBeDefined();

        expect(definition.description).toBeDefined();

        expect(definition).toMatchSnapshot();
    });

});

