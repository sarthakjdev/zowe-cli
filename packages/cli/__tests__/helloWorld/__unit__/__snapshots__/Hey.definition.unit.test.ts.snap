// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hey command definition should say hello 1`] = `
Object {
  "aliases": Array [
    "hw",
  ],
  "description": "hello world command to get familiar with the project, its architecture and working",
  "handler": "/home/sarthak/projects/lfx/zowe-cli/packages/cli/src/helloWorld/Hey.handler",
  "name": "hey",
  "summary": "say hello world",
  "type": "command",
}
`;
